### DISCUSSION TITLE
Suitable substitution for missing JLCPCB components?

### DISCUSSION THREAD

**Discussion by cameleonnzl** [BlueSCSI/BlueSCSI-v2#121] [Build it Yourself]
JLCPCB are short a few components for the Desktop_50_Pin_TopConn_2023.10a gerber. Does anyone know if the following substitutes are suitable?

1. D3: SN74LVTH125PWR (c7042) instead of 19-217/BHC-ZL1M2RY/3T (C72041)
  I read a previous poster had suggested this.
2. U3: AMS1117-3.3 (C2688239) instead of AMS1117-2.85 (C14791)?
  Would the voltage difference be harmful? 3.3v vs 2.85v. Am I better to use AMS1117-2.5 (C2891838)? Or is 2.5v too low?

They're also short of J6 but I've ordered some in as they show available under global sourcing.

**Comment by androda** [comment]
1: D3 is an LED, and part number C7042 is a bus buffer.  They are not interchangeable.  There is no harm in running without D3, that's just the internal drive activity LED.

2: I don't recommend this substitution, the 2.8 to 2.85 volt regulator is a critical voltage.  A substitute part is C2859803, which is similarly out of stock currently.

3: J6, similar to D3, is not critical for device operation.  It's the I2C expansion connector, for which there are no I2C expansions yet.

Parts being in and out of stock is normal for a prototyping provider like JLC.  If you really want to guarantee parts availability, JLC offers a preorder and parts stock system.

**Reply by cameleonnzl** [reply]
Thanks muchly for your advice :) I'll hold off until they either show U3 as in stock, or until I can find a match on a global sourcing search. Already have some for J6 in the order queue when they're back from LNY holiday.
